src/components/Book.js

```diff
@@ -1,32 +1,61 @@
 import './Book.css';
 import React from 'react';
-import styled from 'styled-components';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../thunk';
 
+// const getPercent = `${percentGen}%`;
+
 const Book = (props) => {
   const dispatch = useDispatch();
   const {
     title,
     author,
     category,
     itemId,
+    value,
   } = props;
-
   const submit = () => {
     // const getIDtoDel = event.target.parentElement.parentElement.id;
     // document.getElementById(getIDtoDel).remove();
```

Comment:
- [ ] Same thing here, please check the above ⬆

Reply:
Thank you so much that is very good obervation.